quantum/errors.js

```diff
@@ -39,6 +39,9 @@ const SAME_QUBIT_RECEIVED_TWICE =
 'Please connect the right number of qubits to the node. For circuit output nodes, ' +
 'all qubits should be connected as input. There should be only 1 instance of each qubit at all times in the circuit.';
 
+const NOT_BIT_STRING =
+  'Only bit string consisting 0 and 1 are allowed';
```

Comment:
```suggestion
const NOT_BIT_STRING =
'Only bit string consisting 0 and 1 are allowed';
```
For consistency with the other strings, remove the indentation here.